docs/source/contributing.rst

```diff
@@ -0,0 +1,154 @@
+**********************************
+Contributing to This Documentation
+**********************************
+
+.. highlight:: none
+
+Getting Started
+==========================
+
+This documentation is created using 
+`Sphinx <http://www.sphinx-doc.org/en/stable>`_. Sphinx is an open-source tool 
+that makes it easy to create intelligent and beautiful documentation, written 
+by Georg Brandl and licensed under the BSD license.
+
+The documentation is maintained in the ``master`` branch of the GitHub repository.
+You can include code and its corresponding documentation updates in a single pull-request (PR).
+
+After merging a PR, GitHub Actions automates the documentation building process.
+It pushes the HTML build to the ``gh-pages`` branch, which is hosted on GitHub Pages.
+
+Edit Documentation
+===============================
+
+Sphinx uses `reStructuredText <http://docutils.sourceforge.net/rst.html>`_
+as its markup language. For more information on how to write documentation
+using Sphinx, you can refer to
+
+* `First Steps with Sphinx <http://www.sphinx-doc.org/en/stable/tutorial.html>`_
+* `reStructuredText Primer <http://www.sphinx-doc.org/en/stable/rest.html#external-links>`_
+
+1. Make sure that you have conda in your path. On NERSC machines, you can load it with: ::
+
+   $ module load python/3.7-anaconda-2019.10
+
+2. Clone the repository and checkout a branch from `master`: ::
+
+   $ cd <myDir>
+   $ git clone https://github.com/<your-github-username>/zstash.git
+   $ cd zstash
+   $ git checkout -b <branch-name> master
+
+3. Create and activate the conda development environment ::
```

Comment:
Now using conda dev environment instead of manually creating env and installing sphinx dependencies